ow_plexil/src/plans/README.md

```diff
@@ -60,3 +69,26 @@ The following represent the success criteria of the Reference Mission 2 plan:
 response to a fault
 	- Faults can be injected via rqt, a python script, or the command line
  ([Fault Injection Tutorial](https://github.com/nasa/ow_simulator/blob/master/ow_faults/README.md))
+
+
+### IdentifySampleLocationDemo ###
+
+The following represent the success criteria of the IdentifySampleLocationDemo plan: 
+- In the Rqt image viewer select the /sample_location topic. Once a point is selected
+  the outlined contours of potential sample locations and the chosen target should show up.
+  The green dot represents the selected point.
+- In Rviz to vizualize the 3d point and to see if the guarded move went to the correct location:
+  - Select Add in the left menu under "Displays"
+  - Select "By topic"
+  - Scroll down and select the marker topic /sample_point_visualization and select ok.
```

Comment:
I don't see /sample_point_visualization (see screenshot of menu).  I'm on melodic-devel of ow_simulator.
![rviz-topics](https://user-images.githubusercontent.com/9061371/129595039-19029a2e-3d37-42e5-86a4-65d4358586d8.png)

Reply:
Hmm that is really odd, I am using the melodic-devel branch of ow-simulator. For me it is located at the very bottom if you scroll all the way down:

![MarkerLocation](https://user-images.githubusercontent.com/29695683/129600889-bdf71536-ea93-41d2-a0c9-7c00e69bb228.png)

Is both the main ow launch file and the ow_plexil launch file running?

Reply:
Never mind, I found it, after starting the Plexil node.   This was needed to see the rqt topic as well.  I'm editing the README a bit...